lib/guardian_db/token.ex

```diff
@@ -46,7 +48,7 @@ defmodule GuardianDb.Token do
       |> Map.put("claims", claims)
 
     %Token{}
-    |> cast(prepared_claims, [:jti, :typ, :aud, :iss, :sub, :exp, :jwt, :claims])
+    |> cast(prepared_claims, @allowed_fields)
```

Comment:
@doomspork this line was `81` 😄 so credo complains about it

Reply:
Sounds good @yordis!  We can change the line length to 120, 80 is out-dated in my opinion 👍

Reply:
@doomspork that will require to change `credo` and that is why I dont do it